privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -312,7 +319,66 @@ def actions(cls):
                                 "description": _("Remove a tokengroup from the token."),
                                 "value": [tg.name for tg in get_tokengroups()]
                             }
-                        }
+                        },
+                   ACTION_TYPE.ATTACH_APPLICATION:
+                       {"machine ID":
+                            {
+                                "type": "string",
+                                "required": True,
+                                "description": _("The ID of the machine you want to attach the token to")
+
+                            },
+                        "application":
+                            {
+                                "type": "string",
+                                "required": True,
+                                "description": _("Set a token application like 'offline' or 'SSH'. Note: Not all tokens"
+                                                 " work well with all applications!"),
+                                "value": [TOKEN_APPLICATIONS.SSH, TOKEN_APPLICATIONS.OFFLINE, TOKEN_APPLICATIONS.LUKS]
+                            },
+                        "count":
+                            {
+                                "type": "string",
+                                "visibleIf": "application",
+                                "visibleValue": TOKEN_APPLICATIONS.OFFLINE,
+                                "required": False
+                            },
+                        "rounds":
+                            {
+                                "type": "string",
+                                "visibleIf": "application",
```

Comment:
Please add a description like 

~~~Python
"description": _("The number of rounds for password hashing")
~~~